mitype/app.py

```diff
@@ -161,8 +161,11 @@ def initialize(self, win):
         )
 
         # If required number of lines are more than the window height, exit
-        # +3 for printing stats at the end of the test
-        if self.line_count + 3 > self.window_height:
+        # +5 for printing stats at the end of the test
+        if (
+            self.line_count + 5 >= self.window_height
+            or self.line_count >= self.window_width
```

Comment:
We need not compare with the width right? Line count is the number of lines to fit in given text.

Reply:
In my tests I also found the horizontal error, so the check. But if you prefer, I can remove or simply open another pull request.

Reply:
I agree that mitype will crash when the screen width is not enough to fit in certain elements of the screen. But this value cannot be decided with the help of `line_count`. Basically, the program will crash when there is a large word on the screen, say "encyclopedia" (length = 12), but the screen width is say 10. So, this would require some more work. It is better to create a separate issue for this.